### DISCUSSION TITLE
Nextjs Turbopack server error

### DISCUSSION THREAD

**Discussion by Vatsal-D07** [vercel/next.js#72402] [Help]
### Summary

Error: An unexpected Turbopack error occurred. Please see the output of `next dev` for more details.
    at getServerError (http://localhost:3000/_next/static/chunks/%5Broot%20of%20the%20server%5D__14669f._.js:10403:40)
    at http://localhost:3000/_next/static/chunks/%5Broot%20of%20the%20server%5D__14669f._.js:19091:27

### Additional information

```js
please help me solve this error.
```

**Comment by dev-sujal** [comment]
i got the same error, i created the latest turborepo which hai react 19 and nextjs 15, initially i ran `pnpm run dev` everything done properly, just i added tailwind in `packages/ui` and ran command pnpm install then started getting error on everyfunction like these
![error](https://github.com/user-attachments/assets/591c559d-750d-4937-86b4-3cc3bed6972b)

and getting error -
![error2](https://github.com/user-attachments/assets/a4dd9ecc-fbbc-4cb6-9ac4-f05fb783158b)

while on visiting page getting this error - 
`Error: An unexpected Turbopack error occurred. Please see the output of `next dev` for more details.
`

i tried every possible i can, like reinstalling and configuring turborepo many times, but this error persists everytime i also added types to function like `ReactElement` , `FC` but nothing helped

shifted to old version of turborepo with nextjs 14 and react 18 but there also getting some error and stuck on my project in the middle of development

please any body help to solve this,
although i tried every possible as far as i can but please guide me where i am lacking or this is framework's fault???

**Reply by Daniel-16** [reply]
I got this same error but my fix was a simple `npm install`

**Reply by Felicia-Doumani** [reply]
same

**Comment by luka2220** [comment]
I am also getting this same issue 

![Screenshot 2025-02-25 at 7 55 32 PM](https://github.com/user-attachments/assets/3f06a452-2eca-42dd-b0ee-aed13e69c814)

I am using:
`next: 15.1.7` 
`pnpm: 9.11.0`

OS:
`Mac: macOS Sequoia 15.3.1`

**Comment by FatNerdPeng** [comment]
I have encountered the same problem, and my solution is to delete the dependency lock file under the project. For example, if I am using `pnpm`, I can delete the `pnpm-lock.yaml` file and restart the project successfully. I don't know if this is helpful for you
